pandas/core/series.py

```diff
@@ -251,6 +254,36 @@ class Series(base.IndexOpsMixin, generic.NDFrame):
     Note that the Index is first build with the keys from the dictionary.
     After this the Series is reindexed with the given Index values, hence we
     get all NaN as a result.
+    
+    Constructing Series from an array with `copy=False`.
+    
+    >>> r = [1,2]
+    >>> ser = pd.Series(r, copy=False)
+    >>> ser.iloc[0] = 999
+    >>> r
+    [1, 2]
+    >>> ser
+    0    999
+    1      2
+    dtype: int64
+    
+    The Series returns a `copy` of the original data, so
```

Comment:
What do you mean by returns here? It's more like the data backing the series does not share the same memory as the input `r`.

Reply:
Oh right, by "returns" a copy I meant the Series created had a copy of the data. I guess "returns" is not the correct word to use.